tests/installation/partitioning_raid.pm

```diff
@@ -162,31 +166,38 @@ sub run() {
         assert_screen 'partitioning-size';
         send_key 'ctrl-a';
```

Comment:
ctrl+a = select all text
The selected text is the size in MB. I supposed that it is very unpredictable to make a needle about it.

Can I rely that I will always find the same amount of MB? Is it a default number or a generated amount based on the size of the current disk?